### PR TITLE
Tweak linting in development docs

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

**Comment by ralphtheninja** [comment]
@Simon-Laux Please squash this PR when the time comes for merging :smile:

**Comment by ralphtheninja** [comment]
So essentially we now have only `npm run lint` and `npm run lint:fix-format` that takes care of everything. One thing I noticed was that we have `.jsx` in the list of file types for formatting but we no longer have any of these files right?

File: docs/DEVELOPMENT.md [hunk 1]
```diff
@@ -57,49 +48,38 @@ After making your changes, go in the deltachat/electron Dev-console and press `F
...
 
-We use [ESLint](https://eslint.org) with [TypeScript](https://typescriptlang.org/) rules to catch common bad-practices.
+- [`ESLint`](https://eslint.org) is used with [`TypeScript`](https://typescriptlang.org/) rules to catch common bad practices in all `.js`, `.jsx`, `.ts` and `.tsx` files
+- [`Prettier`](https://prettier.io/) with rules inspired by [`StandardJS`](https://standardjs.com/) to cover formatting in all `.scss`, `.js`, `.ts`, `.tsx`, `.json` and `.md` files
 
 Commands:
 
 ```sh
-# Run to test
+# Check formatting
```
Comment (Simon-Laux) [inline docs/DEVELOPMENT.md:59]: ... and for code problems

File: package.json [hunk 2]
```diff
@@ -5,15 +5,13 @@
...
     "test-misc": "node ./bin/test_package_lock_version",
     "test-log-conventions": "node ./bin/log-conventions",
     "test-unit": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha -r ts-node/register 'test/mocha/**/*.ts'",
     "check-types": "tsc -b src/check-all-tsconfig.json",
-    "check-formatting": "prettier --check \"scss/**/*.scss\" && prettier --check \"src/**/*.{js,ts,tsx,json}\" && prettier --check \"**/*.md\"",
-    "fix-formatting": "prettier --write \"scss/**/*.scss\" && prettier --write \"src/**/*.{js,ts,tsx,json}\" && prettier --write \"**/*.md\"",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "lint:fix-format": "eslint . --ext .js,.jsx,.ts,.tsx --fix || true && npm run fix-formatting",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx && prettier --check \"scss/**/*.scss\" && prettier --check \"src/**/*.{js,ts,tsx,json}\" && prettier --check \"**/*.md\"",
+    "lint:fix-format": "eslint . --ext .js,.jsx,.ts,.tsx --fix || true && prettier --write \"scss/**/*.scss\" && prettier --write \"src/**/*.{js,ts,tsx,json}\" && prettier --write \"**/*.md\"",
```
Comment (Simon-Laux) [inline package.json:14]: we could also call it just `lint:fix`?

**Comment by Simon-Laux** [comment]
> One thing I noticed was that we have .jsx in the list of file types for formatting but we no longer have any of these files right?

yes there are none left of these, we could remove it from the commands

File: package.json (hunk 2 above)
Comment (ralphtheninja) [reply package.json:14]: I wanted to name it this as well but didn't want to change too much. I'll fix!

**Review by Simon-Laux — APPROVED** [review]

**Pull request merged**